pkg/relayer/proxy/websockets/bridge.go

```diff
@@ -115,10 +115,23 @@ func NewBridge(
 	bridgeLogger := logger.With("component", "bridge")
 
 	header := make(http.Header)
+
+	// Add service-specific headers from config
 	for headerKey, headerValue := range serviceConfig.Headers {
 		header.Add(headerKey, headerValue)
 	}
 
+	// Forward identity headers from session and config
+	if session.Header != nil && serviceConfig.ForwardPocketHeaders {
+		// Create a minimal metadata structure to reuse our header-forwarding logic
+		meta := types.RelayRequestMetadata{
+			SessionHeader:           session.Header,
+			SupplierOperatorAddress: relayAuthenticator.GetSupplierOperatorAddresses()[0],
```

Comment:
Can you:

1. Move `relayAuthenticator.GetSupplierOperatorAddresses()[0]` out into a local `supplierOperatorAddr` field
2. Ensure len > 0 before accessing `[0]` -> if not: error
3. If len == 1: easy
4. If len > 1: Warn message saying we're forwarding the `[0]` address

Reply:
Done